experiment/run_experiment.py

```diff
@@ -406,6 +405,7 @@ def start(self):
             '${WORK}/src/experiment/dispatcher.py || '
             '/bin/bash'  # Open shell if experiment fails.
         ]
+        logs.info('Starting dispatcher with container name: %s', container_name)
```

Comment:
nit: "with instance name" to match other message

Reply:
It's in the command.